src/buildTestHarness.tsx

```diff
@@ -142,6 +143,25 @@ export const buildTestHarness =
 
     const typeSpace = async () => type(' ')
 
+    const paste = async (payload: string) =>
+      act(async () => {
+        const event = new window.Event('paste', {
+          bubbles: true,
+          cancelable: true,
+          composed: true,
+        })
+
+        // @ts-ignore Typescript doesn't expect clipboardData on Event type
+        event.clipboardData = {
+          types: ['text/html'],
```

Comment:
I'm not as familiar with this API, but what do you think about adding a second param to this function that that a `type`?

https://w3c.github.io/clipboard-apis/#reading-from-clipboard

```ts
type ClipboardDataType = 'text/html' | 'text/plain' | 'image/png'
const paste = async (payload: string, {types}: {types?: ClipboardDataType[] = ['text/html']}) =>
```

Reply:
Updated!